html/results.html

```diff
@@ -15,8 +15,30 @@
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Dekko&family=Kalam:wght@300;400;700&family=Shadows+Into+Light&display=swap" rel="stylesheet">
   <title>Results</title>
+  
 </head>
 <body>
+  <style>
```

Comment:
@JKrech01 I think if this once becomes the standard for the header and footer, we should put it into the style.css later.